docker-compose.yml

```diff
@@ -19,7 +19,7 @@ services:
       SETTINGS__SURI__URL: http://suri:3000
       SETTINGS__WORKFLOW_URL: http://workflow:3000
       SETTINGS__DOR_SERVICES__URL: http://dor-services-app:3000
-      SETTINGS__DOR_SERVICES__TOKEN: secret-token
+      SETTINGS__DOR_SERVICES__TOKEN: eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJGb28ifQ.-BVfLTW9Q1_ZQEsGv4tuzGLs5rESN7LgdtEwUltnKv4
```

Comment:
Maybe add a comment that we did: `docker-compose run dor-services-app rake generate_token` to make this.